components/poll.js

```diff
@@ -14,6 +15,17 @@ export default function Poll ({ item }) {
   const { me } = useMe()
   const pollVote = usePollVote({ query: POLL_VOTE, itemId: item.id })
   const toaster = useToast()
+  const [pollOptions, setPollOptions] = useState(item.poll.options)
+  const hasShuffled = useRef(false)
+  const shuffleOptions = (options) => {
+    return [...options].sort(() => Math.random() - 0.5)
+  }
```

Comment:
While we're at it, please also move this function outside the component since it doesn't require any context of it.

(I would have done this myself at the end but since I am requesting changes anyway, I decided to mention this here)